histomicstk/segmentation/label/delete_overlap.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+from .condense import condense
+from .delete import delete
+
+
+def delete_overlap(im_label, overlap_info, tile_size):
+    """
+    Deletes overlapping regions from an image label based on overlap information and tile size.
+
+    Args:
+        im_label (ndarray): Image label represented as a NumPy array.
+        overlap_info (dict): Dictionary containing overlap information.
+                            It should have the following keys:
+                            'left', 'right', 'top', and 'bottom',
+                            each specifying the overlap amount in pixels.
+        tile_size (dict): Dictionary specifying the size of the tile.
+                            It should have the following keys:
+                          'height' and 'width', each specifying the tile size in pixels.
+
+    Returns:
+        ndarray: Image label with overlapping regions deleted.
+
+    Note:
+        This function assumes the necessary imports, such as `np`, are already present.
+
+    """
+
+    # Compute the half of the overlap values
+    left_overlap = max(overlap_info['left'] // 2, 0)
+    right_overlap = max(overlap_info['right'] // 2, 0)
+    top_overlap = max(overlap_info['top'] // 2, 0)
+    bottom_overlap = max(overlap_info['bottom'] // 2, 0)
+
+    # Check if the tile size is smaller than 1.5 times the overlap in height or width
+    if tile_size['height'] < overlap_info['top']  \
```

Comment:
I don't understand where this is necessary.  You can't specify an overlap as large as the tile, so there will always be some core tile.  If we are discarding nuclei that are completely in the overlap and the overlap is larger than the largest nuclei, we should never discard too much.  If we discard nuclei that are partially in the outer half of the overlap and the overlap is larger than twice the largest nuclei, we should never discard too much.  Maybe I'm missing something.